PurchasesCoreSwift/PurchasesCoreSwift.h

```diff
@@ -16,4 +16,47 @@ FOUNDATION_EXPORT const unsigned char PurchasesCoreSwiftVersionString[];
 
 // In this header, you should import all the public headers of your framework using statements like #import <PurchasesCoreSwift/PublicHeader.h>
 
+@class RCPurchaserInfo, RCIntroEligibility, RCOfferings;
+@class SKPaymentTransaction, SKProduct, SKPaymentDiscount;
 
+NS_ASSUME_NONNULL_BEGIN
+/**
+ Completion block for calls that send back a `PurchaserInfo`
+ */
+typedef void (^RCReceivePurchaserInfoBlock)(RCPurchaserInfo * _Nullable, NSError * _Nullable);
```

Comment:
we should mark all of these `NS_SWIFT_UNAVAILABLE`, so we're not duplicating the definitions in swift (but with a prefix)

Reply:
✅